ui/client/components/graph/node-modal/editors/expression/Editor.ts

```diff
@@ -56,11 +56,25 @@ export enum EditorType {
   CRON_EDITOR = "CronParameterEditor",
 }
 
+const getValidators = (param: $TodoType) => {
+  if (param == null) { // By default we return mandatory parameter validator
+    return validators[HandledErrorType.MandatoryParameterValidator]()
+  }
+
+  return param.validators.reduce((results: Array<Error>, validator) => {
+    const frontendValidator = validators[validator.type]
+
+    if (frontendValidator instanceof Function) {
+      results.push(frontendValidator())
```

Comment:
why we add Validator to Array<Error>?

Reply:
Upss.. Types in TypeScript don't work very well.. :) I will change it!